### ISSUE TITLE
Vitest not working anymore

### ISSUE THREAD

**Issue by vafanassieff** [webpro/knip#575]
Hey, thanks for this cool tool !

When upgrading from knip `5.1.0` to `5.5.0` or `5.6.0` knip is throwing me an error :

```
❯ npm run knip

> knip-repro-basic@1.0.0 knip
> knip

Analyzing workspace ....
The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
Error: Cannot read properties of null (reading 'startsWith')
    at async Promise.all (index 0)
    at async _0xfa214d._evaluate (https://githubq8ceo8-s054.w-credentialless-staticblitz.com/blitz.d0228c83.js:352:378699)
    at async ModuleJob.run (https://githubq8ceo8-s054.w-credentialless-staticblitz.com/blitz.d0228c83.js:181:2372)
```

Reproduction available [here](https://stackblitz.com/edit/github-q8ceo8)

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.6.1. See [Release 5.6.1](https://github.com/webpro/knip/releases/tag/5.6.1) for release notes._

**Comment by webpro** [comment]
Thanks @vafanassieff, should be fixed now
